platform-common/src/main/java/com/softicar/platform/common/ui/image/Images.java

```diff
@@ -66,4 +68,64 @@ public static List<BufferedImage> readImages(Supplier<InputStream> inputStreamFa
 		}
 		return images;
 	}
+
+	/**
+	 * Determines whether all pixels in the {@link BufferedImage} have the given
+	 * {@link Color}.
+	 *
+	 * @param image
+	 *            the image to probe (never <i>null</i>)
+	 * @param expectedColor
+	 *            the color to probe with (never <i>null</i>)
+	 * @return <i>true</i> if all pixels have the given {@link Color};
+	 *         <i>false</i> otherwise
+	 */
+	public static boolean isSingleColor(BufferedImage image, Color expectedColor) {
+
+		Objects.requireNonNull(image);
+		Objects.requireNonNull(expectedColor);
+
+		int height = image.getHeight();
+		int width = image.getWidth();
+
+		for (int x = 0; x < width; x++) {
+			for (int y = 0; y < height; y++) {
+				if (!new Color(image.getRGB(x, y)).equals(expectedColor)) {
```

Comment:
could be premature optimization but `if(image.getRGB(x, y) != expectedColor.getRGB())` might be more efficient

Reply:
Yea... probably. Changed accordingly.